src/ApiService/ApiService/onefuzzlib/orm/Queries.cs

```diff
@@ -46,12 +46,25 @@ public static string And(IEnumerable<string> queries)
             return EqualAny(property, convertedEnums);
         }
 
+        public static string EqualEnum<T>(string property, T e) where T : Enum {
+            string convertedEnum = JsonSerializer.Serialize(e, EntityConverter.GetJsonSerializerOptions()).Trim('"');
+            return $"{property} eq '{EscapeString(convertedEnum)}'";
+        }
+
         public static string TimeRange(DateTimeOffset min, DateTimeOffset max) {
             // NB: this uses the auto-populated Timestamp property, and will result in a table scan
             // TODO: should this be inclusive at the endpoints?
             return TableClient.CreateQueryFilter($"Timestamp lt {max} and Timestamp gt {min}");
         }
 
+        public static string NewerThan(DateTimeOffset t) {
+            return TableClient.CreateQueryFilter($"Timestamp gt {t}");
```

Comment:
```suggestion
            return TableClient.CreateQueryFilter($"Timestamp gt {t.ToString("o")}");
```

To keep the same behavior from `TimerRetention`

Reply:
TableClient.CreateQueryFilter <- does the conversion based on type. So it's supposed to do t.ToString("o") implicitly